docs/src/tutorial.md

```diff
@@ -0,0 +1,241 @@
+# Tutorial
+
+In this tutorial we can see the main uses of `SolverBenchmark`.
+
+First, let's create fake data. It is imperative that the data for each solver is stored
+in `DataFrame`s, and the collection of different solver must be stored in a dictionary of
+`Symbol` to `DataFrame`.
+
+In our examples we'll use the following data.
+
+```@example ex1
+using DataFrames, Printf, Random
+
+Random.seed!(0)
+
+n = 10
+names = [:alpha, :beta, :gamma]
+stats = Dict(name => DataFrame(:id => 1:n,
+         :name => [@sprintf("prob%03d", i) for i = 1:n],
+         :status => map(x -> x ? :success : :failure, rand(n) .< 0.75),
```

Comment:
This syntax is odd. I would have written `map(x -> x < 0.75 ? :success : :failure, rand(n))`.